packages/logger/src/helpers/helper-simple-deep-clone.ts

```
@@ -0,0 +1,51 @@
+      output[i] = simpleDeepClone(input[i]);
+    }
+    return output;
+  }
+  if (isClassInstance(input)) {
+    return input;
+  }
+  // handle case: object
+  output = {};
+  for (i in input) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (input.hasOwnProperty(i)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)